internal/pkg/util/fs/mount/mount.go

```diff
@@ -339,7 +348,8 @@ func (p *Points) add(tag AuthorizedTag, source string, dest string, fstype strin
 			}
 		}
 		if present {
-			return fmt.Errorf("destination %s is already in the mount point list", dest)
+			return ErrMountExists
+
```

Comment:
Remove this empty line here. I would suggest that actually a new blank line can be added after the next `}` character (add empty line after line 353)